data/stoa0040/stoa074/__cts__.xml

```diff
@@ -2,7 +2,7 @@
   <ti:title xml:lang="lat">Quaestiones in Heptateuchum</ti:title>
   <ti:edition  xml:lang="lat" workUrn="urn:cts:latinLit:stoa0040.stoa074" urn="urn:cts:latinLit:stoa0040.stoa074.opp-lat1">
     <ti:label xml:lang="lat">Quaestiones in Heptateuchum</ti:label>
-    <ti:description xml:lang="mul">Augustine.  Sancti Aureli Augustini Opera, Sectio III, Pars 3. 
-      Zycha, Joseph, editor. Corpus scriptorum ecclesiasticorum Latinorum, Volume 28.2, 1896.</ti:description>
+    <ti:description xml:lang="mul">Augustine.  Sancti Aureli Augustini Opera, Sectio III, Pars 3. (Corpus scriptorum ecclesiasticorum Latinorum, Volume 28.2.)
+      Zycha, Joseph, editor. Prague; Vienna; Leipzig: F. Tempsky; G. Freytag, 1895..</ti:description>
```

Comment:
duplicate period here

Reply:
I fixed this so I'm going to merge this pull request since the other file looks ok.